packages/table/src/utils/style.css

```diff
@@ -0,0 +1,10 @@
+div.iot-table-cell {
+  display: flex;
+  align-items: center;
+}
+
+div.iot-table-cell div.icon {
```

Comment:
don't need to specify that it's a `div`, just apply styles to the classes

Reply:
removed